docs/reference/configuration/keywords.rst

```diff
@@ -257,25 +254,27 @@ NUM_CPU
 .. _num_cpu:
 
 
-This keyword is set right in your configuration file:
+This keyword tells the compute system (LSF/Torque/Slurm) how many cpus/cores
+each realization needs.
+
+*Example:*
 
 .. code-block:: none
 
-  NUM_CPU 42
+  NUM_CPU 4
 
-Its meaning varies depending on context. For LSF it equates to the ``-n``
-parameter. See more here https://www.ibm.com/support/knowledgecenter/SSWRJV_10.1.0/lsf_command_ref/bsub.n.1.html.
-E.g. ``NUM_CPU 10`` can be understood as a way for a job to make sure it can
-execute on ``10`` processors. This means that a higher number may *increase*
-wait times, since LSF might need to wait until resources are freed in order to
-allocate 10 processors.
+Note that if you are using Eclipse and the :ref:`DATA_FILE <data_file>` keyword,
+this is implicitly set. If you need to override, use ``NUM_CPU`` explicitly.
 
-For TORQUE, it literally is a check that ``NUM_CPU`` is larger than the amount
-of resources TORQUE wants to allocate (number_of_nodes \* cpus_per_node). See
-:ref:`NUM_NODES|NUM_CPUS_PER_NODE <torque_nodes_cpus>` for details.
+This number should be regarded as informational to the queueing system and is
+likely to affect scheduling, in that a realization will not start until
+sufficient CPU resources are assumed available. It will not guarantee CPU
+exclusiveness on the compute node. If set incorrectly, there is no guarantee
+that the realization will fail.
```

Comment:
```suggestion
exclusiveness on the compute node. If set incorrectly, there is no guarantee
that the realization will run.
```
Is this correct?

Reply:
Nope. You are lucky if you are able to fail your realization by specifying wrong cpu count. We had only a few failures when num_cpu was missing altogheter in the lsf_driver and hundreds of multi-cpu Eclipse runs were running and all of them said to the LSF system what "I only need 1 cpu".